src/core/trtserver.h

```diff
@@ -281,6 +281,13 @@ TRTSERVER_EXPORT TRTSERVER_Error*
 TRTSERVER_ServerOptionsSetTensorFlowGpuMemoryFraction(
     TRTSERVER_ServerOptions* options, float fraction);
 
+// Add Tensorflow virtual GPU instances to physical GPU. Specify limit
+// for total memory available for use on that physical GPU.
+TRTSERVER_EXPORT TRTSERVER_Error*
+TRTSERVER_ServerOptionsAddTensorFlowVgpuMemoryLimits(
+    TRTSERVER_ServerOptions* options, int gpu_device, int num_vgpus,
+    float mem_limit);
```

Comment:
Memory should be per-vgpu memory, so how about name it "per_vgpu_memory_mbytes". And it should be a uint64_t, not a float.